tests/systems/slurm/test_installer.py

```
@@ -368,3 +368,17 @@ def test_mark_as_installed_local_container(slurm_system: SlurmSystem):
     installer.mark_as_installed_one(docker_image)
 
     assert docker_image.installed_path == local_image.absolute()
+
+
+def test_mark_as_installed_duplicate_local_containers_preserve_local_path(slurm_system: SlurmSystem):
+    installer = SlurmInstaller(slurm_system)
+
+    local_image = Path("/tmp/local_image.sqsh")
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/local_image.sqsh"

(S108)